app/src/main/kotlin/com/wire/android/ui/common/imagepreview/BulletHoleImagePreview.kt

```diff
@@ -44,6 +48,19 @@ import androidx.constraintlayout.compose.ConstraintLayout
 import coil.compose.rememberAsyncImagePainter
 import com.wire.android.ui.common.dimensions
 import com.wire.android.util.toBitmap
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+@Composable
+private fun loadBiMap(imageUri: Uri): State<Bitmap?> {
```

Comment:
Question 1: Why is this change on this PR? 🤔 
Question 2/nitpick: 
```suggestion
private fun loadBitMap(imageUri: Uri): State<Bitmap?> {
```

Reply:
We were calling` imageUri.toBitmap(context)` on main thread